tests/components/totalconnect/test_config_flow.py

```diff
@@ -163,4 +160,9 @@ async def test_reauth(hass):
         assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
         assert result["reason"] == "reauth_successful"
 
+    with patch(
```

Comment:
I'd move this patch to the same place as the TotalConnectClient patch, to be on the safe side.